package.json

```diff
@@ -55,6 +55,7 @@
     }
   },
   "scripts": {
+    "axe-rule-gen": "node bin/rule-generator",
```

Comment:
Not sure we need to prefix this with `axe-`. When you run it, you'll be doing so from within the axe-core repo, so it's probably a little redundant.

Reply:
It does make sense from a global context...but do we really need that? I don't see much of a reason to have this exist globally since the feature only applies to this repo.